tests/integration/test_integration_authenticator_state.py

```diff
@@ -57,6 +73,30 @@ async def test_disable_non_existing_authenticator(self):
             response = await c.set_authenticator_state('authn-iam/non-existing', False)
         self.assertEqual(context.exception.status, 401)
 
+    async def test_authenticator_enable_without_service_id_success(self):
+        c = await create_client(self.valid_user.id, self.valid_user.api_key)
+        response = await c.set_authenticator_state('authn-gcp', True)
+
+        self.assertEqual(response, '')
+
+    async def test_authenticator_disable_without_service_id_success(self):
+        c = await create_client(self.valid_user.id, self.valid_user.api_key)
+        response = await c.set_authenticator_state('authn-gcp', False)
+
+        self.assertEqual(response, '')
+
+    async def test_authenticator_enable_without_service_id_failure(self):
```

Comment:
Maybe it will be clearer if the test name mentioned that the failure is because the invalid user and not because there is no service id

Reply:
Good point. Fixed it